### PR TITLE
Deleted mail subdomain and Changed Email

### PR DESCRIPTION
**Pull request by sekiryl**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://l6174.is-a.dev

### CODE REVIEW

File: domains/l6174.json [hunk 1]
```diff
@@ -3,9 +3,11 @@
   "repo": "https://github.com/l6174/l6174.github.io",
   "owner": {
     "username": "l6174",
-    "email": "l.lawliet6174@gmail.com"
+    "email": "l6174@proton.me"
   },
   "record": {
-    "CNAME": "l6174.github.io"
+    "CNAME": "l6174.github.io",
```
Comment (wdhdev) [inline domains/l6174.json:9]: You cannot use a CNAME record with other records.

**Review by wdhdev — DISMISSED** [review]

File: domains/l6174.json (hunk 1 above)
Comment (l6174) [reply domains/l6174.json]: I am left with no choice but to remove MX and SPF records. Is there any other alternative BTW?

Comment (CuteDog5695) [reply domains/l6174.json]: > I am left with no choice but to remove MX and SPF records. Is there any other alternative BTW?

You know you can also use A records for Github Pages, right?

Comment (l6174) [reply domains/l6174.json]: > > I am left with no choice but to remove MX and SPF records. Is there any other alternative BTW?
> 
> You know you can also use A records for Github Pages, right?

Isn't that only for Apex domains?

Comment (CuteDog5695) [reply domains/l6174.json]: It works also on subdomains (a lot of users have been doing this)

**Review by iostpa — APPROVED** [review]
Seems good to me now

File: domains/l6174.json (hunk 1 above)
Comment (l6174) [reply domains/l6174.json]: Ok I will try that and open another PR. Thanks!

**Pull request merged**